list/PageableList.js

```diff
@@ -716,47 +650,36 @@ define([
 			return function () {
 				sup.call(this, arguments);
 				if (this.pageLength > 0) {
-					this._nextPageLoader = null;
-					this._previousPageLoader = null;
+					this._showNextPageLoader = false;
+					this._showPreviousPageLoader = false;
 					this._rangeSpec = null;
 					this._untrack();
 					this._firstLoaded = this._lastLoaded = -1;
+					this._dataLoaded = false;
 				}
 			};
 		}),
 
-		_getNextRenderer: dcl.superCall(function (sup) {
-			//	make sure that no page loader is returned
-			return function (renderer, /*jshint unused:vars*/dir) {
-				var value = sup.apply(this, arguments);
-				if ((this._nextPageLoader && value === this._nextPageLoader)
-					|| (this._previousPageLoader && value === this._previousPageLoader)) {
-					value = null;
-				}
-				return value;
-			};
-		}),
-
 		_spaceKeydownHandler: dcl.superCall(function (sup) {
 			//	Handle action key on page loaders
 			return function (event) {
-				if (this._nextPageLoader && this._nextPageLoader.contains(event.target)) {
+				if (this._showNextPageLoader && this.nextPageLoader.contains(event.target)) {
 					event.preventDefault();
-					this._nextPageLoader._load();
-				} else if (this._previousPageLoader && this._previousPageLoader.contains(event.target)) {
+					this._load(event);
+				} else if (this._showPreviousPageLoader && this.previousPageLoader.contains(event.target)) {
 					event.preventDefault();
-					this._previousPageLoader._load();
+					this._load(event);
 				} else {
 					sup.apply(this, arguments);
 				}
 			};
 		}),
-		
+
 		handleSelection: dcl.superCall(function (sup) {
 			// page loader should never be selected when clicked
 			return function (event) {
-				var renderer = this.getEnclosingRenderer(event.target);
-				if (renderer === this._nextPageLoader || renderer === this._previousPageLoader) {
+				if (this._showPreviousPageLoader && this.previousPageLoader.contains(event.target) ||
+					this._showNextPageLoader && this.nextPageLoader.contains(event.target)) {
 					return;
 				} else {
 					sup.apply(this, arguments);
```

Comment:
I don't think you need this method at all.

Rather, shouldn't List.js be changed so instead of:

```
this.on("click", this.handleSelection.bind(this));
```

it does that for `this.containerNode`?   You could put `on-click={{_handleSelection}}` in the template.

Having said that, I don't know how keyboard selection is working, so you should investigate that.